conf/test_full.config

```
@@ -23,6 +23,7 @@ params {
     genome = 'R64-1-1'
 
     // parameters
-    skip_eukulele = true
-    skip_eggnog   = true
+    skip_eukulele  = true
+    skip_eggnog    = true
+    skip_kofamscan = true
```

```suggestion
    skip_eukulele  = false
    skip_eggnog    = false
    skip_kofamscan = false
```
I think we should actually have all parts running in this test case.